.github/workflows/run-tests.yml

```diff
@@ -1,39 +1,46 @@
 name: Run tests
```

Comment:
Please revert file changes and only add `if: "!contains(github.event.head_commit.message, 'skipci')"` and update versions numbers of `checkout` and `setup-php`